### PR TITLE
chore: add missing information to price database landing page

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves https://artsyproduct.atlassian.net/browse/CX-1838

**Description:**
This PR populates the price database landing page with the missing data

https://user-images.githubusercontent.com/11945712/134497838-64ec0f8c-8399-4577-be34-ef91adb24398.mov

Thank you @olerichter00

### CODE REVIEW

File: src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx [hunk 1]
```diff
@@ -52,51 +70,204 @@ export const PriceDatabaseBenefits: React.FC = () => {
...
+      <Image
+        width="100%"
+        height="100%"
+        src={src}
+        srcSet={srcSet}
+        alt={alt}
+        style={{ objectFit: "cover", alignSelf: "center" }}
+      />
+      {!!artistName && !!artworkName && (
+        <>
```
Comment (AlicanAkyuz) [inline src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx:130]: small chore: this fragment seems to be not needed here.

**Review by AlicanAkyuz — APPROVED** [review]
LGTM!

File: src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx:130]: oh yes, i refactored this and forgot to update it. I will fix it now. Thanks

**Comment by olerichter00** [comment]
#squashongreen

**Review by olerichter00 — APPROVED** [review]

**Pull request merged**